improver/spotdata/apply_lapse_rate.py

```diff
@@ -48,7 +48,11 @@ class SpotLapseRateAdjust(PostProcessingPlugin):
     orography.
     """
 
-    def __init__(self, neighbour_selection_method: str = "nearest") -> None:
+    def __init__(
+        self,
+        neighbour_selection_method: str = "nearest",
+        fixed_lapse_rate: float = None,
+    ) -> None:
         """
         Args:
```

Comment:
`fixed_lapse_rate` needs describing in this doc-string

Reply:
Done.

Reply:
Should we specify the units of this float? Or is it assumed to be in the units of the input data?

Reply:
It's assuming SI units K/m. I've written this into the doc string now, here and in the CLI.